docs/sparc.client.rst

```diff
@@ -0,0 +1,29 @@
+sparc.client package
+====================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   sparc.client.services
+
+Submodules
+----------
+
+sparc.client.client module
+--------------------------
```

Comment:
Heading should be on another level?

Reply:
Not sure if I understand. 
Would you like me to remove "sparc.client" part @hsorby, or something different?

Reply:
I think that this heading should be a sub-heading of the `submodules` heading, not at the same level as it.

Reply:
Is this what you mean @hsorby ? 
Link to docs: https://athril.github.io/sparc.client/
If so, please feel free to merge.

Reply:
No, that isn't what I meant.  You have not made the heading a different level from `Submodules`.

Reply:
I adjusted the indentation manually, is it fine now @hsorby ?
For the record, by doing this we are moving away from automatic documentation generation towards more hands-on approach. Previously, I used `sphinx-apidoc -o . ../src` to generate templates, but the current version requires manual adjustments.